pkg/api/core/shoot/warnings_test.go

```diff
@@ -43,6 +43,7 @@ var _ = Describe("Warnings", func() {
 			shoot = &core.Shoot{
 				Spec: core.ShootSpec{
 					Kubernetes: core.Kubernetes{
+						Version:                     "1.22.11",
```

Comment:
Not sure why you modified this, but it actually brings me to an idea: Let's write a user-facing warning if the `Shoot` is 1.23+ and still does not disable the PSP admission plugin. This way, end-users can be made aware already before they plan upgrading to 1.25.

Reply:
Yes, this exactly was the plan, but since now we don't allow to disable this plugin (it's in the requiredAdmissionPlugin list), it won't make sense to print the warning now. After all the extensions are released, I will add this warning back.

Reply:
Cool, can you add a respective task for this to https://github.com/gardener/gardener/issues/5250 so that we don't lose track/forget about this? :)

Reply:
Done.